### PR TITLE
moved canceling `.later` timer to separate function

### PR DESCRIPTION
**Pull request by bekzod**

### CODE REVIEW

File: lib/backburner/utils.ts [hunk 1]
```diff
@@ -52,9 +52,9 @@ export function findItem(target, method, collection) {
 export function findTimer(timer, collection) {
   let index = -1;
 
-  for (let i = 0, l = collection.length; i < l; i += 3) {
-    if (collection[i + 2] === timer) {
```
Comment (bekzod) [inline lib/backburner/utils.ts]: micro-optimization avoid additional incrementation

File: lib/index.ts [hunk 2]
```diff
@@ -679,22 +685,21 @@ export default class Backburner {
...
     let timers = this._timers;
-    let l = timers.length;
     let defaultQueue = this.options.defaultQueue;
     let n = now();
-    for (; i < l; i += 2) {
+
+    for (let i = 1; i < timers.length; i += 2) {
       let executeAt = timers[i];
       if (executeAt <= n) {
-        let fn = timers[i + 1];
```
Comment (bekzod) [inline lib/index.ts]: micro-optimization avoid additional incrementation

**Comment by bekzod** [comment]
this brought slight benchmark improvements

**Comment by stefanpenner** [comment]
@bekzod thanks for this attention to detail!

**Pull request merged**